js/runtime.js

```diff
@@ -100,7 +101,7 @@
             incrementVariable: function(variable, value){
                 this[name] += value;
             },
-            loopOver: function(args, containers){
+            loopOver: function(args, containers) {
```

Comment:
Haha! I have no idea why that got changed. Maybe it happened while I was fixing jshint's warnings about tabs and spaces (jshint automatically runs whenever I save a file).